system/dummy_diag_publisher/package.xml

```diff
@@ -11,6 +11,7 @@
 
   <depend>diagnostic_updater</depend>
   <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
```

Comment:
Could you write what error has happened by this, in the description section? :pray:

Reply:
@kenji-miyake Sure!